Lectures/Lecture3_BigCollaborations.md

```diff
@@ -1,41 +1,62 @@
+- Course participants have critically reflected on challenges faced by large collaborations and choices to be made in their structure, organization, and membership.
+- Course participants have investigated examples of working within large collaborations, in particular the credit structures and  ways to promote junior researchers. 
 
 ## Embedding/community
-__please mention the main community that should be considered when studying the topics of this lecture, e.g. individual researchers, research community, employers, general public, etc. __
+This lecture focuses on large national as well as international collaborations. Many of the large collaborations involve interdisciplinary components, both within physics and with engineering and industry. See https://bigscience.nl/about for a concerted effort in NL to foster synergies between physics and technology in large collaborations. 
 
 ## Methods/exercises
-**Topics covered in lecture**
-+ topic 1
-+ topic 2
++ In preparation for this lecture, course participants are asked to watch the two-part video lecture by Dr Niels Martens on the epistemology of scientific collaborations. Lecture 1 of this set (https://www.youtube.com/watch?v=M1giIxeLuIg) discusses examples of 'big science' projects such as CERN and the Next Generation Event Horizon Telescope. Lecture 2 of this set (https://www.youtube.com/watch?v=DRt1uTytWf8) focuses on ways to manage the large number of co-authors that are involved in these projects and policies for giving appropriate credit. 
++ Course participants will work in small groups to investigate the credit structure of one collaboration they were assigned from the list below. They prepare a short presentation to their peers. 
++ The groups will give their presentations to the class and have a final critical discussion on the topic of large collaborations, for which Dr. Niels Martens will also join. 
 
 ## Evaluation
-__assignments or evaluation of the desired learning outcome__
++ Students answer the quiz about the video lectures by Dr. Niels Martens. 
++ Students work in small groups to investigate an example of the credit structure and promotion of junior researchers within a large collaboration.
++ Students give a short presentation of their findings to the class, receive peer feedback, and hear from the others on different examples.  
++ Students participate in the final discussion. 
 
 ### Assignment description
 
+Students work in small groups to investigate the credit structure of one of the large collaborations from the list below. They prepare a short presentation that addresses:
+
+1. A brief introduction to the collaboration, its scientific aims, when it was established, and its membership. 
+2. An assessment of how credit structures are implemented and how junior researchers can grow within this big collaboration and gain visibility in the field, as far as can be determined from the resources that are made publicly available by the collaboration. 
+
```

Comment:
we also discussed to add an investigation of the open science skills and practices emphasized in the collaboration. Can we add that also as a third topic for the investigation?